app/views/form/_select_question.html.erb

```diff
@@ -0,0 +1,11 @@
+
+<%= answers = question["answer_options"].map {|key, value| OpenStruct.new(id:key, name: value)}
```

Comment:
Needs linting I think

Reply:
(Spaces on the `}`)

Reply:
ran rubocop on it. Didn't show any errors.

Reply:
Interesting, wonder if it doesn't lint erbs